provider/pkg/gen/python-templates/helm/v2/helm.py

```diff
@@ -406,8 +429,15 @@ class ChartOpts(BaseChartOpts):
     Additional options to customize the fetching of the Helm chart.
     """
 
-    def __init__(self, chart, namespace=None, values=None, transformations=None, resource_prefix=None, repo=None,
-                 version=None, fetch_opts=None):
+    def __init__(self,
+                 chart: pulumi.Input[str],
+                 namespace: Optional[pulumi.Input[str]] = None,
+                 values: Optional[pulumi.Inputs] = None,
+                 transformations: Optional[pulumi.Inputs] = None,
```

Comment:
Should we be more specific here? The docstring looks a bit different than this.

Reply:
Good catch. Copy/paste mistake. Will fix.